app/models/observation.rb

```diff
@@ -314,6 +314,10 @@ class Observation < AbstractModel
         ->(name) { where(name: Name.text_name_includes(name)) }
   scope :by_user,
         ->(user) { where(user: user) }
+  scope :mappable,
+        -> { where.not(location: nil).or(where.not(lat: nil)) }
+  scope :unmappable,
+        -> { where(location: nil).and(where(lat: nil)) }
```

Comment:
Is this sort of like pork in a congressional bill?  Throwing a bone to a special interests group? :)

Reply:
:)